corehq/apps/locations/views.py

```diff
@@ -834,6 +834,16 @@ def location_export(request, domain):
     return response
 
 
+def get_locations_paylod_from_ids(ids, domain, base_queryset):
+    from corehq.apps.locations.util import get_locations_from_ids
+    locations = get_locations_from_ids(ids, domain, base_queryset=base_queryset)
+    return [loc_to_payload(loc) for loc in locations]
+
+
+def base_queryset_for_domain_and_user(domain, user):
+    return SQLLocation.objects.accessible_to_user(domain, user)
```

Comment:
It just occurred to me that we should make `accessible_to_user` always filter results by domain.  For non-restricted users, this currently doesn't add a domain filter, and I could see things like this confusingly including results from other domains, if that filter isn't explicitly added elsewhere.